src/python/pants/pantsd/pants_daemon.py

```diff
@@ -312,6 +312,17 @@ def _run_services(self, services):
       self._logger.info('starting service {}'.format(service))
       try:
         service_thread.start()
+        if isinstance(service, SchedulerService):
```

Comment:
This is slightly awkward, but seems less fragile than the hidden path construction from before. Thanks!

Reply:
Yeah, how do we generally deal with getting service instances? I would've assumed that we would've had a map of {class: instance} or something on self, but we seem to just have a tuple... 

Or am I expected to just grab a global singleton here or something?